From: Roman Lebedev <lebedev.ri@gmail.com>
Date: Sat, 27 Jul 2024 06:31:53 +0300
Subject: Dissable LTO for python stub

`Halide_PyStubs` static library is installed,
but if we build Halide with LTO, it, obviously,
contains Clang/LLVM IR representation,
not assembly code, so things go awry.

Forwarded: not-needed
---
 python_bindings/stub/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/python_bindings/stub/CMakeLists.txt b/python_bindings/stub/CMakeLists.txt
index 9a6a2c8..e918b8a 100644
--- a/python_bindings/stub/CMakeLists.txt
+++ b/python_bindings/stub/CMakeLists.txt
@@ -1,3 +1,5 @@
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION OFF)
+
 add_library(Halide_PyStubs STATIC ${CMAKE_CURRENT_LIST_DIR}/PyStubImpl.cpp)
 add_library(Halide::PyStubs ALIAS Halide_PyStubs)
 
